keyboards/kbdcraft/adam64/adam64.c

```diff
@@ -0,0 +1,97 @@
+/* 
+ * Copyright 2023 KBDCraft
+ * Copyright 2023 Adophoxia <andyao1528@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "quantum.h"
+
+const is31_led PROGMEM g_is31_leds[RGB_MATRIX_LED_COUNT] = {
```

Comment:
```suggestion
#ifdef RGB_MATRIX_ENABLE
const is31_led PROGMEM g_is31_leds[RGB_MATRIX_LED_COUNT] = {
```

Reply:
Added the `#ifdef RGB_MATRIX_ENABLE` block.